extensions/community/UpdateChecker.json

```diff
@@ -0,0 +1,273 @@
+  "name": "UpdateChecker",
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/update.svg",
+  "shortDescription": "Checks if the version of the game on liluo is different than the current game version.",
+  "version": "1.0.0",
+  "tags": [
+    "update",
+    "liluo"
+  ],
+  "authorIds": [
+    "ZgrsWuRTAkXgeuPV9bo0zuEcA2w1"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [
+    {
+      "description": "",
+      "fullName": "",
+      "functionType": "Action",
+      "group": "",
+      "name": "onFirstSceneLoaded",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "value": "UpdateChecker::RecheckForUpdates"
+              },
+              "parameters": [
+                "",
+                ""
+              ]
+            }
+          ]
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Loads the latest game version from liluo. This is already called at the beginnign of the scene, but you can still re-call it if it failed or if you want to add a \"check again\" button. Make sure to not call this too often (or at all), as otherwise it will DDoS the version-checking server!.",
+      "fullName": "Re-check for updates",
+      "functionType": "Action",
+      "group": "",
+      "name": "RecheckForUpdates",
+      "private": false,
+      "sentence": "Request latest game version from liluo",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "value": "UpdateChecker::CreateGlobalReference"
+              },
+              "parameters": [
+                "",
+                "__UpdateChecker.LatestVersion",
+                "__UpdateChecker.LatestVersion",
+                ""
+              ]
+            },
+            {
+              "type": {
+                "value": "UpdateChecker::CreateGlobalReference"
+              },
+              "parameters": [
+                "",
+                "__UpdateChecker.Error",
+                "__UpdateChecker.Error",
+                ""
+              ]
+            },
+            {
+              "type": {
+                "value": "SendAsyncRequest"
+              },
+              "parameters": [
+                "\"https://updates.arthuro555.com/?game-id=\" + UpdateChecker::CurrentGameID()",
```

Comment:
Can you explain this request to your server, please?

Also, a request on a server always raises the question of its maintenance and availability.